### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by anushka-maheshwari**

### CODE REVIEW

**Comment by anushka-maheshwari** [comment]
[NodeJs & Express Basics](https://trello.com/c/6anDVJlk/73-nodejs-express-basics)

**Comment by anushka-maheshwari** [comment]
[NodeJs & Express Basics](https://trello.com/c/6anDVJlk/73-nodejs-express-basics)

File: USER_SYSTEM/router.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const express=require("express");
+const router=express.Router();
+const credentials={
+    email:"admin@gmail.com",
```
Comment (JainPriya6) [inline USER_SYSTEM/router.js:4]: Never store the credentials or any sensitive data directly in the code. Always use the `.env` file for the same.

File: USER_SYSTEM/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "user_system",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (JainPriya6) [inline USER_SYSTEM/package.json:8]: node "start" script seems to be missing.

**Review by JainPriya6 — APPROVED** [review]
1. Always add a meaningful description to the commits when pushing the code. Just adding 'Initial Commit' or 'Final Commit' is not a good practice.
2. Improve upon the other two reviews added above in the files for further app practices. Rest is good. PR can be merged.

**Pull request merged**